static/js/filters/number.js

```diff
@@ -0,0 +1,18 @@
+angular.module('inboxFilters').filter('maxInteger', function() {
```

Comment:
Why do we need this filter? AFAICT it's only used in JS code that could just use `Math.max`